[2023-12-20T11:48:17.103+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2021-11-15T00:00:00+00:00 [queued]>
[2023-12-20T11:48:17.120+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: our_first_dag_v2.second_id scheduled__2021-11-15T00:00:00+00:00 [queued]>
[2023-12-20T11:48:17.121+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 6
[2023-12-20T11:48:17.147+0000] {taskinstance.py:2192} INFO - Executing <Task(BashOperator): second_id> on 2021-11-15 00:00:00+00:00
[2023-12-20T11:48:17.154+0000] {standard_task_runner.py:60} INFO - Started process 5247 to run task
[2023-12-20T11:48:17.159+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'our_first_dag_v2', 'second_id', 'scheduled__2021-11-15T00:00:00+00:00', '--job-id', '1198', '--raw', '--subdir', 'DAGS_FOLDER/first_dat.py', '--cfg-path', '/tmp/tmpiuld1ytj']
[2023-12-20T11:48:17.164+0000] {standard_task_runner.py:88} INFO - Job 1198: Subtask second_id
[2023-12-20T11:48:17.250+0000] {task_command.py:423} INFO - Running <TaskInstance: our_first_dag_v2.second_id scheduled__2021-11-15T00:00:00+00:00 [running]> on host 29b3723064ac
[2023-12-20T11:48:17.402+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='abrham' AIRFLOW_CTX_DAG_ID='our_first_dag_v2' AIRFLOW_CTX_TASK_ID='second_id' AIRFLOW_CTX_EXECUTION_DATE='2021-11-15T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='scheduled__2021-11-15T00:00:00+00:00'
[2023-12-20T11:48:17.404+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2023-12-20T11:48:17.404+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo hey, I am task2 and will be runing after task 3']
[2023-12-20T11:48:17.421+0000] {subprocess.py:86} INFO - Output:
[2023-12-20T11:48:17.424+0000] {subprocess.py:93} INFO - hey, I am task2 and will be runing after task 3
[2023-12-20T11:48:17.425+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2023-12-20T11:48:17.464+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=our_first_dag_v2, task_id=second_id, execution_date=20211115T000000, start_date=20231220T114817, end_date=20231220T114817
[2023-12-20T11:48:17.537+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2023-12-20T11:48:17.572+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
